docs/overview/getting-started/avax.md

```diff
@@ -1,26 +1,79 @@
 # AVAX
 
-AVAX is the native utility token of Avalanche. It’s a hard-capped, scarce asset that is used to 
+AVAX is the native utility token of Avalanche. It’s a hard-capped, scarce asset that is used to
 pay for fees, secure the platform through staking, and provide a basic unit of account between the
 multiple Subnets created on Avalanche. `1 nAVAX` is equal to `0.000000001 AVAX`.
 
 ## Utility
 
-AVAX is a non-renewable resource (analogous to oil) that's used to power the Avalanche network.
-AVAX is used for day-to-day operations in Avalanche such as staking and issuing transactions.
+AVAX is a capped-supply (up to 720M) resource in the Avalanche ecosystem that's used to power the
+network. AVAX is used to secure the ecosystem through staking and for day-to-day operations like
+issuing transactions.
 
-AVAX represents the weight that each node has in network decisions. No single actor owns 
-the Avalanche Network, so each validator in the network is given a proportional weight in the 
-network's decisions corresponding to the proportion of total stake that they own.
+AVAX represents the weight that each node has in network decisions. No single actor owns
+the Avalanche Network, so each validator in the network is given a proportional weight in the
+network's decisions corresponding to the proportion of total stake that they own through proof
+of stake (PoS).
 
-Any entity trying to execute a workload on Avalanche pays a corresponding fee (commonly known as 
-"gas") to run it on the network. The gas used to execute a workload on Avalanche is removed from
-existence (commonly known as "burning"). 
+Any entity trying to execute a transaction on Avalanche pays a corresponding fee (commonly known as
+"gas") to run it on the network. The fees used to execute a transaction on Avalanche is burned,
+or permanently removed from circulating supply.
 
-A small portion of burnt gas fees are used to reward validators at the end of their staking 
-periods. Validators are rewarded for virtuous (e.g not attempting lie, steal, or otherwise
-cause harm the Avalanche network) behavior by receiving validator rewards at the end of their
-staking periods. Avalanche does not take away any portion of a validator's already staked tokens
-(commonly known as "slashing") for negligent/malicious staking periods, however this behavior is
-disincentivized as validators who attempt to do harm to the network would expend their node's
-computing resources for no reward.
+## Tokenomics
+
+A fixed amount of 360M AVAX was minted at genesis, but a small amount of AVAX is constantly minted
+as a reward to validators. Validators are rewarded for virtuous (e.g not attempting to lie, steal, or
+otherwise cause harm the Avalanche network) behavior by receiving minted validator rewards at the
+end of their staking periods. The minting process offsets the process of burning and prevents the
```

Comment:
This last sentence is a little misleading. That doesn't actually solve the "scarce AVAX" problem you bring up. I'd suggest: `The minting process offsets the AVAX burned by transactions fees. While AVAX is still far away from its supply cap, it will almost always remain an inflationary asset.`

Reply:
Thanks, reworded to this.